launch_testing_ros/launch_testing_ros/wait_for_topics.py

```diff
@@ -86,6 +93,12 @@ def topics_not_received(self):
         """Topics that did not receive any messages."""
         return self.__ros_node.expected_topics - self.__ros_node.received_topics
 
+    def messages_received(self, topic_name):
+        """List of received messages of a specific topic."""
+        if topic_name not in self.__ros_node.received_messages:
+            raise KeyError("No message received with topic " + topic_name)
+        return list(self.__ros_node.received_messages[topic_name])
```

Comment:
Though the ``__ros_node`` is meant as a private member, ``received_messages`` the dictionary, and ``messages_received`` the method can be confusing to read. It'd be nice to differentiate the container and the method, maybe using the ``_`` prefix or renaming it to something like ``received_messages_buffer`` and the method as ``received_messages``

Reply:
Thank you for the suggestion. I fixed it in commit 0f4753521b74f8505b17a98b496f7bfee60279a0.